### ISSUE TITLE
YT Setup video is set to private

### ISSUE THREAD

**Issue by makergamer** [HakCat-Tech/WiFi-Nugget#6]
The "Quickstart Setup: https://youtu.be/WAG7yCbEFtw" YT Video is set to Private so it is unavailable to those with a WiFi-Nugget.

**Comment by doukkani17moha** [comment]
> The "Quickstart Setup: https://youtu.be/WAG7yCbEFtw" YT Video is set to Private so it is unavailable to those with a WiFi-Nugget.

Yes me too, there is another video: https://www.youtube.com/watch?v=L7Avqv_-CTo&t=307s
